compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt

```diff
@@ -1091,8 +1091,10 @@ internal class AccessibilityMediator(
             while (true) {
                 invalidationChannel.receive()
 
+                // Wait until all changes in the accessibility tree have been completed
+                // and consume all invalidations
+                delay(1)
```

Comment:
Is it 1s or 1ms?

Reply:
Delay api always takes ms. Added label for clarity.